src/model/documents/ingredient.ts

```diff
@@ -0,0 +1,11 @@
+import { BaseModel } from '../base-model';
+import { UnitOfMeasure } from '../enums/unit-of-measure';
+import { Category } from './category';
+
+export interface Ingredient extends BaseModel {
+  imagePath: string;
+  name: string;
+  category: Category;
```

Comment:
Why we setup category for ingredients? I think it is enough just for recipe/articles.

Reply:
Referring to the documentation - it should be there.
![image](https://user-images.githubusercontent.com/48626088/113477530-a9c74400-9482-11eb-8e90-8a5070831b42.png)